docs/styles.md

```diff
@@ -2,14 +2,39 @@
 
 In order to maintain consistency and reliability across all RAISE content, the styling associated with RAISE content is consolidated into a series of css classes located in `src/styles`. 
 
-RAISE Style classes are either written for specific elements, or can be applied to a variety of objects. Those differences are listed below.
+RAISE Style classes are either written for specific elements, or can be applied to a variety of objects. Those differences are listed below. Content Developers 
 
-## Style Classes 
 
-### Indent (os-raise-indent)
+## Gray Box (os-raise-graybox)
+Add a Gray box around your element
+
+**Example**
+<div style="text-align: center;">
+    <img src="./static/graybox.png" width="450">
```

Comment:
Should we update the screenshot to not include the styled `<p>`? (or I think that's included here at least)